impl_core/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -298,13 +303,19 @@ void resumeStatsCollection(Timestamp now) {
 
     private IntervalMutableViewData(View view, Timestamp start) {
       super(view);
-      Duration totalDuration = ((Interval) view.getWindow()).getDuration();
-      this.totalDuration = totalDuration;
-      this.bucketDuration = Duration.fromMillis(toMillis(totalDuration) / N);
+      View.AggregationWindow.Interval window = (View.AggregationWindow.Interval) view.getWindow();
 
-      // When initializing. add N empty buckets prior to the start timestamp of this
-      // IntervalMutableViewData, so that the last bucket will be the current one in effect.
-      shiftBucketList(N + 1, start);
+      if (window == null) { // work around checker framework
+        throw new AssertionError();
```

Comment:
If `Window` is null, we won't create an `IntervalMutableViewData` in the first place. Anyway this change is reverted since `Window` should continue to be NonNull.